src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -86,26 +96,40 @@ const TopicPage = ({ pageData }) => {
             {description && <TopicDescription>{description}</TopicDescription>}
           </div>
           {curations.map(
-            ({
-              visualProminence,
-              summaries,
-              curationId,
-              title: curationTitle,
-              link,
-              position,
-            }) => (
-              <Curation
-                headingLevel={curationTitle && 3}
-                key={curationId}
-                visualStyle={VISUAL_STYLE.NONE}
-                visualProminance={visualProminence}
-                promos={summaries}
-                title={curationTitle}
-                topStoriesTitle={topStoriesTitle}
-                position={position}
-                link={link}
-                curationLength={curations && curations.length}
-              />
+            (
+              {
+                visualProminence,
+                summaries,
+                curationId,
+                title: curationTitle,
+                link,
+                position,
+                visualStyle,
+              },
+              index,
+            ) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <React.Fragment key={`${curationId} ${index}`}>
```

Comment:
I just had a thought - we can use `position` instead of `index` here, as `position` will always be unique for each curation in the list. (Please note, we might need to fix linting if we commit the suggestion in GitHub)
```suggestion
            ) => (
              <React.Fragment key={`${curationId}-${position}`}>
```